pages/docs/image.mdx

```diff
@@ -0,0 +1,351 @@
+---
+title: Image
+---
+
+import IIIFBadge from "docs/components/IIIFBadge";
+import CloverImage from "docs/components/DynamicImports/Image";
+import { Tabs, Tab } from "nextra/components";
+
+export const handleOpenSeadragonCallback = (viewer) => {
+  console.log("OpenSeadragon instance is ready");
+  console.log(viewer);
+};
+
+# Image
+
+A UI component the renders a pan and deep-zoom Image viewer using [OpenSeadragon](https://openseadragon.github.io/).
+
+<IIIFBadge href="https://iiif.io/api/image/3.0" text={["Image"]} />
+
+<div style={{ height: "38.2vh", minHeight: "450px" }}>
+  <CloverImage
+    body={{
+      id: "https://iiif.dc.library.northwestern.edu/iiif/2/6ca016c5-de7f-4373-ae8f-7047fecf6ace/full/600,/0/default.jpg",
+      type: "Image",
+      format: "image/tiff",
+      height: 5709,
+      width: 8949,
+      service: [
+        {
+          "@id":
+            "https://iiif.dc.library.northwestern.edu/iiif/2/6ca016c5-de7f-4373-ae8f-7047fecf6ace",
+          "@type": "ImageService2",
+          profile: "http://iiif.io/api/image/2/level2.json",
+        },
+      ],
+    }}
+    openSeadragonCallback={handleOpenSeadragonCallback}
+  />
+</div>
+
+## Features
+
+Provide a IIIF Annotation body with the `type` of `Image` and the component:
+
+- Renders a single [OpenSeadragon](https://openseadragon.github.io/) instance.
+- Returns OpenSeadragon's `Viewer` instance for programmatic control.
+- Provides configuration options for OpenSeadragon's `Viewer` instance.
+
+## Installation
+
+<Tabs items={["npm", "yarn", "pnpm"]}>
+  {/* prettier-ignore */}
+  <Tab>
+    ```shell
+      npm install @samvera/clover-iiif
+    ```
+  </Tab>
+  {/* prettier-ignore */}
+  <Tab>
+    ```shell
+      yarn add @samvera/clover-iiif
+    ```
+  </Tab>
+  {/* prettier-ignore */}
+  <Tab>
+    ```shell
+      pnpm install @samvera/clover-iiif
+    ```
+  </Tab>
+</Tabs>
+
+## Usage
+
+### React
+
+Add the `Image` component to your `jsx` or `tsx` code.
+
+```jsx
+import Image from "@samvera/clover-iiif/image";
```

Comment:
This is a great pattern, nice to see it spread around.  A consuming app doesn't have to pull down the entire Clover suite. 👍